docs/content/users/install/docker-installation.md

```diff
@@ -64,7 +64,7 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
     * If you already have Docker Desktop installed, make sure to disable its integration with your WSL2 distro. In *Resources* → *WSL Integration*, disable integration with the default distro and with your particular distro.
     * If you don’t already have WSL2, install it with `wsl --install`. This will likely require a reboot.
     * Run `wsl --set-default-version 2`.
-    * Install a distro. We recommend Ubuntu 20.04: `wsl -s Ubuntu-20.04`.
+    * Install a distro. We recommend Ubuntu 20.04: `wsl --install Ubuntu-20.04`.
```

Comment:
Actually, if you'd done `wsl --install` earlier, this wouldn't have been necessary at all. It installs Ubuntu (which is currently 20.04). So this line should be omitted. However, the MS flow is kind of unpredictable. If you have already installed WSL2 then `wsl --install` doesn't work this way, and you have to `wsl --install Ubuntu`

Reply:
```suggestion
    * If the `Ubuntu` distro is not already installed and the default distro (see `wsl -l -v`) then install it with `wsl --install Ubuntu`.
```